### PR TITLE
RWA-286 update initiate dmn table

### PR DESCRIPTION
**Pull request by iguannago**
https://tools.hmcts.net/jira/browse/RWA-286

### CODE REVIEW

**Review by lewis150193 — APPROVED** [review]

File: src/main/resources/wa-task-initiation-ia-asylum.dmn [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <definitions xmlns="https://www.omg.org/spec/DMN/20191111/MODEL/" xmlns:camunda="http://camunda.org/schema/1.0/dmn" xmlns:biodi="http://bpmn.io/schema/dmn/biodi/2.0" xmlns:dmndi="https://www.omg.org/spec/DMN/20191111/DMNDI/" xmlns:dc="http://www.omg.org/spec/DMN/20180521/DC/" id="wa-initiation-definition" name="DRD" namespace="http://camunda.org/schema/1.0/dmn" exporter="Camunda Modeler" exporterVersion="4.2.0">
   <decision id="wa-task-initiation-ia-asylum" name="Task initiation DMN">
-    <decisionTable id="DecisionTable_0jtevuc" hitPolicy="FIRST">
+    <decisionTable id="DecisionTable_0jtevuc" hitPolicy="COLLECT">
       <input id="Input_1" label="Event Id" camunda:inputVariable="eventId">
         <inputExpression id="InputExpression_1" typeRef="string">
           <text></text>
```
Comment (BenNzewi) [inline src/main/resources/wa-task-initiation-ia-asylum.dmn:7]: Process Application is missing task category mapping  -> "Case progression"

File: src/integrationTest/java/uk/gov/hmcts/reform/iataskconfiguration/CamundaTaskInitiationTest.java [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+        "submitCase, caseUnderReview, reviewAppealSkeletonArgument, TCW, 2, true, Case progression",
+        "submitReasonsForAppeal, reasonsForAppealSubmitted, reviewReasonsForAppeal, TCW, 2, true, Case progression",
+        "submitClarifyingQuestionAnswers, clarifyingQuestionsAnswersSubmitted, reviewClarifyingQuestionsAnswers, TCW, 2, true, Case progression",
+        "submitCmaRequirements, cmaRequirementsSubmitted, reviewCmaRequirements, TCW, 2, true, Case progression",
+        "listCma, cmaListed, attendCma, TCW, 2, true, Case progression",
+        "uploadHomeOfficeAppealResponse, respondentReview, reviewRespondentResponse, TCW, 2, true, Case progression",
+        "anything, prepareForHearing, createCaseSummary, TCW, 2, true, Case progression",
+        "anything, finalBundling, createHearingBundle, TCW, 2, true, Case progression",
+        "anything, preHearing, startDecisionsAndReasonsDocument, TCW, 2, true, Case progression",
+        "draftHearingRequirements, listing, reviewHearingRequirements, TCW, 2, true, Case progression, "
```
Comment (BenNzewi) [inline src/integrationTest/java/uk/gov/hmcts/reform/iataskconfiguration/CamundaTaskInitiationTest.java:53]: Can we include a scenario for "Review the Appeal"?

File: src/integrationTest/java/uk/gov/hmcts/reform/iataskconfiguration/CamundaTaskInitiationTest.java [hunk 3]
```diff
@@ -0,0 +1,293 @@
...
+    void setUp() {
+        dmnEngine = DmnEngineConfiguration
+            .createDefaultDmnEngineConfiguration()
+            .buildEngine();
+    }
+
+    @DisplayName("Get task id")
+    @ParameterizedTest(name = "\"{0}\" \"{1}\" should go to \"{2}\"")
+    @CsvSource({
+        "submitAppeal, anything, processApplication, TCW, 2, false, ",
```
Comment (BenNzewi) [inline src/integrationTest/java/uk/gov/hmcts/reform/iataskconfiguration/CamundaTaskInitiationTest.java:41]: "submitAppeal, anything, processApplication, TCW, 2, false, " ->should check task category for processApplication is Case Progression.

File: src/main/resources/wa-task-initiation-ia-asylum.dmn (hunk 1 above)
Comment (iguannago) [reply src/main/resources/wa-task-initiation-ia-asylum.dmn:7]: I raised this question with Helen. Will update it accordingly.

File: src/integrationTest/java/uk/gov/hmcts/reform/iataskconfiguration/CamundaTaskInitiationTest.java (hunk 3 above)
Comment (iguannago) [reply src/integrationTest/java/uk/gov/hmcts/reform/iataskconfiguration/CamundaTaskInitiationTest.java:41]: I raised this question with Helen. Will update it accordingly.

File: src/integrationTest/java/uk/gov/hmcts/reform/iataskconfiguration/CamundaTaskInitiationTest.java (hunk 2 above)
Comment (iguannago) [reply src/integrationTest/java/uk/gov/hmcts/reform/iataskconfiguration/CamundaTaskInitiationTest.java:53]: yes, I will add it.

File: src/integrationTest/java/uk/gov/hmcts/reform/iataskconfiguration/CamundaTaskInitiationTest.java (hunk 3 above)
Comment (iguannago) [reply src/integrationTest/java/uk/gov/hmcts/reform/iataskconfiguration/CamundaTaskInitiationTest.java:41]: Helen says to leave it as it is

File: src/main/resources/wa-task-initiation-ia-asylum.dmn (hunk 1 above)
Comment (iguannago) [reply src/main/resources/wa-task-initiation-ia-asylum.dmn:7]: Helen says to leave it as it is

File: src/integrationTest/java/uk/gov/hmcts/reform/iataskconfiguration/CamundaTaskInitiationTest.java (hunk 2 above)
Comment (iguannago) [reply src/integrationTest/java/uk/gov/hmcts/reform/iataskconfiguration/CamundaTaskInitiationTest.java:53]: this is done here https://github.com/hmcts/ia-task-configuration/pull/44/files

**Pull request merged**